docs/index.md

```diff
@@ -3,85 +3,326 @@ hide:
+    <a class="section-button" href="./components/input">
+      Form components
+      <div class="section-button-subtext">e,g. input, checkbox, radio</div>
+    </a>
+    <a class="section-button" href="./components/table">
+      Data display components
+      <div class="section-button-subtext">e,g. table, plot</div>
+    </a>
+  </div>
+</section>
+
+<section class="container teal-gradient-background">
+  <div class="section-chip teal">Build anything</div>
+  <div class="section-headline">
+    Build any user interface you can imagine
+  </div>
+  <div class="section-body">
+With Mesop, you can build virtually any web-based user interface or application you can imagine. From quick prototypes to enterprise tools, Mesop provides the customizability to bring your ideas to life.
+  </div>
+  <div class="center">
+  <!-- UPDATE: -->
+  <iframe src="http://localhost:32123/embed/chat_inputs" class="chat-inputs-iframe box-shadow"></iframe>
+  </div>
+</section>
+
+<section class="container amber-gradient-background">
+  <div class="section-chip amber">Extensible</div>
+  <div class="section-headline">
+  Seamlessly integrate JS with web components
+  </div>
+  <div class="section-body">
+With Mesop <a href="./web_components/">web components</a>, you get the best of both worlds: the simplicity and power of Python, with the ability to drop down into JavaScript when you need that extra control.  Use existing JS libraries - all while keeping your core logic in Python.
+  </div>
+</section>
+
+<section class="container blue-gradient-background">
+  <div class="section-chip blue">Simple deployment</div>
+  <div class="section-headline">
+    Deploy your app and share in minutes
+  </div>
+  <div class="section-body">
+Mesop streamlines cloud deployment, enabling you to share your AI application with the world in minutes. With step-by-step guides for deploying to Google Cloud Run or any cloud service that takes a container, you can go from local development to production-ready deployment without wrestling with complex server setups.
+<a class="section-button" href="./guides/deployment" style="display: flex">
+<img src="https://www.gstatic.com/bricks/image/f2e0986a2802c0b6c4be7f1355599d5aadfb21a63b7e9643d96697ff9334a1e1.svg" style="width: 40px; margin-right: 12px">
+  <div>
+      Google Cloud Run
+      <div class="section-button-subtext">Free for small apps</div>
+      </div>
+    </a>
+  </div>
+</section>
+
+<section class="container pink-gradient-background">
+  <div class="section-chip pink">Developer experience</div>
+  <div class="section-headline">
+    Delightful developer experience
+  </div>
+  <div class="section-body">
+Mesop streamlines app development with features like hot reload and strong IDE support with static types, eliminating friction and boosting productivity.
+</div>
+  <div class="dev-gif-row">
+    <div class="gif-caption">
+      Instant hot reload
+    </div>
+    <img src="./assets/hot_reload.gif" class="dev-gif box-shadow">
```

Comment:
[nit] Minor thing, but maybe record an example without a bunch of tabs open. Though I guess that's more realistic.